### DISCUSSION TITLE
macos hide native title bar buttons

### DISCUSSION THREAD

**Discussion by zzuiekwei** [AvaloniaUI/Avalonia#12904] [Q&A]
how do i hide the minimize and full screen buttons native to macos ?

**Comment by timunie** [comment]
Have you tried different `SystemDecorations="None"` already? I don't know if Mac does allow that 🤔 at all.

**Reply by zzuiekwei** [reply]
SystemDecorations="None" will hide all the buttons,  but i want hide not all of them. for example , the window can only be closed, so only show the close button, and the minisize and fullscreen button hide.
